website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -34,10 +34,10 @@ compatible Envoy versions.
 
 | Consul Version      | Compatible Envoy Versions        |
 | ------------------- | -------------------------------- |
-| 1.10.x              | 1.17.0, 1.16.2, 1.15.3, 1.14.6‡, |
-| 1.9.x               | 1.16.2, 1.15.3, 1.14.6‡, 1.13.7‡ |
-| 1.8.x               | 1.14.6, 1.13.7, 1.12.7, 1.11.2   |
-| 1.7.x               | 1.13.7, 1.12.7, 1.11.2, 1.10.0\* |
+| 1.10.x              | 1.18.2, 1.17.2, 1.16.3, 1.15.4   |
+| 1.9.x               | 1.16.3, 1.15.4, 1.14.7‡, 1.13.8‡ |
+| 1.8.x               | 1.14.7, 1.13.8, 1.12.7, 1.11.2   |
+| 1.7.x               | 1.13.8, 1.12.7, 1.11.2, 1.10.0\* |
```

Comment:
The manual backports of this pr to the older series will finish making these 3 lines true.